CppImport/src/ClangAstConsumer.cpp

```diff
@@ -46,4 +41,11 @@ void ClangAstConsumer::setCompilerInstance(const clang::CompilerInstance* compil
 	astVisitor_->setPreprocessor(&compilerInstance->getPreprocessor());
 }
 
+void ClangAstConsumer::HandleTranslationUnit(clang::ASTContext& astContext)
+{
+	astVisitor_->TraverseDecl(astContext.getTranslationUnitDecl());
+
+	astVisitor_->macroImportHelper_.macroGeneration();
```

Comment:
In general, you should use verbs for method names. Here, without having read the rest of the code, I feel like `generateMacros` is more appropriate, or perhaps even `generateMetaCodeFromMacros`, depending on what you're doing.